llrt_modules/src/lib.rs

```diff
@@ -11,3 +12,7 @@ pub mod stream;
 #[cfg(test)]
 mod test;
 mod utils;
+
+pub const VERSION: &str = env!("CARGO_PKG_VERSION");
+
+pub static TIME_ORIGIN: AtomicUsize = AtomicUsize::new(0);
```

Comment:
I think we need to have the discussion on that static at https://github.com/awslabs/llrt/issues/503